MobSF/utils.py

```diff
@@ -628,11 +628,9 @@ def get_proxy_ip(identifier):
 
 def is_safe_path(safe_root, check_path):
     """Detect Path Traversal."""
-    safe_root = os.path.normpath(safe_root)
-    check_path = os.path.normpath(check_path)
-    return os.path.commonprefix(
-        (os.path.realpath(check_path), safe_root)) == safe_root
-
+    safe_root = os.path.realpath(os.path.normpath(safe_root))
+    check_path = os.path.realpath(os.path.normpath(check_path))
+    return os.path.commonprefix((check_path, safe_root)) == safe_root
```

Comment:
Please remove extra brackets. it's not required.

Reply:
If you are talking about this:
```suggestion
    return os.path.commonprefix(check_path, safe_root) == safe_root
```

Then it is actually a tuple, so it is required. I first removed and checked, it wasn't working.

Reply:
It should be a list or path-like object. See: https://docs.python.org/3/library/os.path.html#os.path.commonprefix

Reply:
Okay so should I change the tuple to a list?
```suggestion
    return os.path.commonprefix([check_path, safe_root]) == safe_root
```

Reply:
Right